packages/baguette_sharp/baguette_sharp.2.2.1/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "The Baguette# Interpreter REPL"
+description: "The REPL for Baguette#"
+maintainer: ["Charlotte Thomas"]
+authors: ["Charlotte Thomas"]
+license: "GPLv3"
```

Comment:
Your `LICENSE` file seems to indicate this is `GPL-3.0-or-later` (according to the SPDX standard)
```suggestion
license: "GPL-3.0-or-later"
```